src/acvp.h

```diff
@@ -1380,6 +1451,77 @@ ACVP_RESULT acvp_enable_dsa_cap_parm (ACVP_CTX *ctx,
                                       ACVP_DSA_PARM param,
                                       int value);
 
+/*! @brief acvp_enable_kas_ecc_cap()
+
+  This function should be used to enable KAS-ECC capabilities. Specific modes
+  and parameters can use acvp_enable_kas_ecc_cap_parm.
+
+   When the application enables a crypto capability, such as KAS-ECC, it
+   also needs to specify a callback function that will be used by libacvp
+   when that crypto capability is needed during a test session.
+
+   @param ctx Address of pointer to a previously allocated ACVP_CTX.
+   @param cipher ACVP_CIPHER enum value identifying the crypto capability.
+   @param crypto_handler Address of function implemented by application that
+      is invoked by libacvp when the crypto capability is needed during
+      a test session.
+
+   @return ACVP_RESULT
+*/
+ACVP_RESULT acvp_enable_kas_ecc_cap (
+        ACVP_CTX *ctx,
+        ACVP_CIPHER cipher,
+        ACVP_RESULT (*crypto_handler) (ACVP_TEST_CASE *test_case));
+
+/*! @brief acvp_enable_kas_ecc_prereq_cap() allows an application to specify
+        a prerequisite algorithm for a given KAS-ECC mode during a test session
+        with the ACVP server.
+
+        This function should be called to enable a prerequisite for
+        an KAS-ECC mode capability that will be tested by the server.
+
+   @param ctx Address of pointer to a previously allocated ACVP_CTX.
+   @param cipher ACVP_CIPHER enum value identifying the crypto capability.
+   @param mode ACVP_KAS_ECC_MODE enum value specifying mode. An example would be
+        ACVP_KAS_ECC_MODE_PARTIAL
+   @param pre_req ACVP_PREREQ_ALG enum that the specified cipher/mode
+        depends on
+   @param value "same" or number
+
+   @return ACVP_RESULT
+ */
+ACVP_RESULT acvp_enable_kas_ecc_prereq_cap (
```

Comment:
Does this one have to separate from the general enable_prereq API?

Reply:
Unfortunately it does since the prereqs are in the mode(s) not the cap